src/main/java/de/qabel/desktop/ui/accounting/AccountingController.java

```diff
@@ -82,4 +89,66 @@ protected void addIdentityWithAlias(String alias) {
 			clientConfiguration.selectIdentity(identity);
 		}
 	}
+
+	@FXML
+	protected void handleImportIdentityButtonAction(ActionEvent event) {
+
+		FileChooser chooser = new FileChooser();
+		chooser.setTitle("Choose Download Folder");
+		File file = chooser.showOpenDialog(identityList.getScene().getWindow());
+		try {
```

Comment:
everything below line 99 should be extracted to get a test (then only the FileChooser part is untested)